CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ the code was deployed.
 ### Security
 
 - Upgrade dependencies (CU-860phzbq5).
+- Add API Key to webhooks (CU-yfdndf).
```

Comment:
I'd specify in here that this is the Particle webhooks because it's possible that other third party systems that we interact with also use webhooks.